README.md

```diff
@@ -1,2 +1,170 @@
+- [📝 License](#license)
+
+## 📖 Blog App
+
+<a name="about-project"></a>
+
+**The Blog App** is a classic example of a blog website. It is a fully functional website that shows the list of posts and empower readers to interact with them by adding comments and liking posts.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Ruby
+- Ruby on Rails
+- VSCode
+- Rubocop
+- PostgreSQL
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Linters</summary>
+  <ul>
+  <li>StyleLint</li>
+  <li>Rubocop</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+  <li>Ruby</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+  <li>PostgreSQL</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add posts**
+- **Add comments**
+- **Add like posts**
+
+<p align="right">(<a href="#readme-top">back to the top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo Not Available
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  git clone https://github.com/teke85/blog-app.git
+  cd blog_app
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd blog-app
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec spec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+👤 **Isiteketo mutau**
+
+- GitHub: [@teke85](https://github.com/teke85)
+- Twitter: [@muttau](https://twitter.com/muttau)
+- LinkedIn: [isiteketo mutau](https://www.linkedin.com/in/isiteketo-mutau-736894241/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add Authorization rules]**
+- [ ] **[Add API endpoints.]**
+- [ ] **[Integration specs for Views and fixing n+1 problems.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/teke85/blog-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a star!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./license) licensed.
```

Comment:
- [ ] Please fix the [LICENSE](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) link on your README file the current link is broken.

- Change the file path on the LICENSE link

- Give it your LICENSE file name (MIT.md)

Reply:
Hi @CollinsTatang I have fixed the link to my readme file thank you for taking time to review my project